scm-ui/ui-components/src/Tooltip.tsx

```diff
@@ -31,9 +31,11 @@ type Props = {
   children: ReactNode;
 };
 
+export type TooltipLocation = "bottom" | "right" | "top" | "left";
```

Comment:
Why is the type not used for the location prop?

Reply:
I was not sure if this would be a breaking change. I know it shouldn't be but it kinda seems like one.

Reply:
Changing a type definition is not a breaking change. Type are not relevant for the runtime at all. They are only hints for the developer and this change makes the hint only better and does not change anything at runtime.